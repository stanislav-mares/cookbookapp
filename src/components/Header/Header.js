import React from 'react';
import PropTypes from 'prop-types';

//images
import logo from 'images/cookHat4.png'

const Header = (props) => {
  return(
    <header>
      <div>
        <img src={logo} width="50px" height="50px" />
        <span>Cookbook application</span>
      </div>
    </header>
  );
}

Header.propTypes = {

};

export default Header;
