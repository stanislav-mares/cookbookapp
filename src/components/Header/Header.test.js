import React from 'react';
import enzymeConfig from 'testConfig/enzymeConfig';
import { shallow } from 'enzyme';

import Header from 'components/Header/Header';

describe('<Header />', () => {
  it('Testing structure of the Header component', () => {
    const component = shallow(<Header />);

    expect(component).toMatchSnapshot();
  });
});
