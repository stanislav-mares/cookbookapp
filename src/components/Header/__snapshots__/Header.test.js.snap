// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Testing structure of the Header component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <div>
        <img
          height="50px"
          src="test-file-stub"
          width="50px"
        />
        <span>
          Cookbook application
        </span>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            height="50px"
            src="test-file-stub"
            width="50px"
          />,
          <span>
            Cookbook application
          </span>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "height": "50px",
            "src": "test-file-stub",
            "width": "50px",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Cookbook application",
          },
          "ref": null,
          "rendered": "Cookbook application",
          "type": "span",
        },
      ],
      "type": "div",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <div>
          <img
            height="50px"
            src="test-file-stub"
            width="50px"
          />
          <span>
            Cookbook application
          </span>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              height="50px"
              src="test-file-stub"
              width="50px"
            />,
            <span>
              Cookbook application
            </span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "height": "50px",
              "src": "test-file-stub",
              "width": "50px",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": "Cookbook application",
            },
            "ref": null,
            "rendered": "Cookbook application",
            "type": "span",
          },
        ],
        "type": "div",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
